Finishlinelab3/finishlinelab3infra.yaml

```diff
@@ -48,6 +50,26 @@ Resources:
           FromPort: '22'
           ToPort: '22'
           CidrIp: !Ref SSHLocation
+  CharityDBServer:
+    Type: 'AWS::EC2::Instance'
+    Properties:
+      InstanceType: !Ref InstanceType
+      SecurityGroups:
+        - !Ref InstanceSecurityGroup
```

Comment:
You will have to reference both Web SG and DB SG

` - !Ref CharityWebServerSG
  - !Ref CharityDBSG`

Reply:
done

Reply:
done